BioscoopSysteemAPI/BioscoopSysteemAPI/Services/TemplateMethodSeatAssignment.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace BioscoopSysteemAPI.Services
```

Comment:
Dit document vervangt SeatAssignment toch? 

Als dit zo is zou ik SeatAssignment.cs verwijderen om geen dubbele code te hebben.

Reply:
Ja klopt als jullie die ook beter vinden dan de voorgaande versie. Dan verwijderen ik de oude.